homeassistant/const.py

```diff
@@ -815,41 +815,117 @@ class UnitOfSpeed(StrEnum):
 SIGNAL_STRENGTH_DECIBELS: Final = "dB"
 SIGNAL_STRENGTH_DECIBELS_MILLIWATT: Final = "dBm"
 
+
 # Data units
+class UnitOfData(StrEnum):
```

Comment:
In general, this is referred to as "Units of Information"

See:
- https://en.wikipedia.org/wiki/File_size
- https://en.wikipedia.org/wiki/Units_of_information